src/gui/transferlistfilterswidget.cpp

```diff
@@ -70,11 +70,14 @@ namespace
         return !scheme.isEmpty() ? scheme : u"http"_qs;
     }
 
-    QString getHost(const QString &tracker)
+    QString getHost(const QString &str)
```

Comment:
Isn't it expected that `getHost()` takes URL as parameter? Wouldn't it be better to use `url` as parameter name?

Reply:
Done.